ansible/scripts/deploy-local-cluster.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
I got feeling that is valid on kubernetes/kubernetes repository, not in the contrib one.